registry/storage/driver/rados/rados.go

```diff
@@ -471,6 +471,7 @@ func (d *driver) Delete(ctx context.Context, objectPath string) error {
 		totalSize, err := d.getXattrTotalSize(ctx, oid)
 
 		if err != nil {
+			err = d.deleteOid(objectPath)
```

Comment:
Sure, but do you also intend to hide the error, as well? This sets `err` to `nil` if the delete succeeds, making it look like the whole operation proceeded.